packages/components/src/public/QueryModel/TabbedGridPanel.tsx

```diff
@@ -60,7 +60,11 @@ export interface TabbedGridPanelProps<T = {}> extends GridPanelProps<T> {
      */
     activeModelId?: string;
     /**
-     * By default if there is only one model, the tabs will not be shown.  Setting this to true will show the tab
+     * Defaults to true. Determines if we allow columns to be hidden and rearranged.
```

Comment:
since this eventually will be more things like "adding columns, etc.", I suggest just making the comment generic like "Determines if we allow the view to be customized (add/hide columns, reorder columns, etc.)"